test/debug-puppetieer.js

```
@@ -0,0 +1,37 @@
+  await page.emulateMediaFeatures([{ name: 'prefers-color-scheme', value: 'dark' }])
+
+  // Navigate the page to a URL.
+  const storyId = 'components-rangeslider--default'
+  await page.goto(
+    `http://localhost:6006/iframe.html?globals=backgrounds.value%3A!hex(111)&args=&id=${storyId}&viewMode=story`,
+    {
+      waitUntil: 'networkidle0',
+    }
+  )
+
+  debugger
```

Biome (2.1.2)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)